### PR TITLE
nlmixr2, NONMEM and lag time

### PR DESCRIPTION
**Pull request by mattfidler**

### CODE REVIEW

**Comment by mattfidler** [comment]
@kestrel99 ready to review and/or edit or commit

**Comment by kestrel99** [comment]
> @kestrel99 ready to review and/or edit or commit

Hi! What is the full citation for Gill, Murray, Saunders, and Wright (1983)?

**Comment by mattfidler** [comment]
P. E. Gill, W. Murray, M. A. Saunders, and M. H. Wright, Computing forward-difference
intervals for numerical optimization, SIAM Journal on Scientific and Statistical Comput-
ing, 4 (1983), pp. 310–321.

**Comment by mattfidler** [comment]
I couldnt find it ehen I wrote it but it was cited in th shi 2021 paper too

File: content/blog/2022-11-10-nlmixr2-nonmem/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+## New method for handling dose-based sensitivities
+
+Now in `nlmixr2`, we use the method proposed in Shi 2021
+(https://doi.org/10.48550/arXiv.2110.06380).  This method is extended
+so that instead of optimizing the gradient of one rate, we take the
+harmonic mean of all the calculated rates for each observation and
+optimize the best step size for that value (which is the same method
+we use for the Hessian calculation in the generalized log-likelihood).
+
+In our limited testing of lag-times, this seems to perform better than
```
Comment (billdenney) [inline content/blog/2022-11-10-nlmixr2-nonmem/index.Rmd:80]: I'd suggest removing the word "limited" from here.

Comment (mattfidler) [reply content/blog/2022-11-10-nlmixr2-nonmem/index.Rmd:80]: Thats fine with me.

Comment (billdenney) [reply content/blog/2022-11-10-nlmixr2-nonmem/index.Rmd:80]: I made that change, but when I tried to regenerate the html everything changed.  I think that someone else (or automation) will need to update the html.

**Comment by mattfidler** [comment]
You need to use blogdown

I always use blogdown::check_site() and it tella me what to do.

**Pull request merged**